docs/zcml.rst

```diff
@@ -1129,6 +1131,10 @@ First we provide a stub configuration context:
 .. doctest::
 
    >>> import re, pprint
+   >>> try:
+   ...     from cStringIO import StringIO
+   ... except ImportError:
+   ...     from io import StringIO
```

Comment:
You can't 😢 The `PrettyPrinter` calls `stream.write("(")` which is a str (aka bytes on Python 2).